db/levelsConvert.yml

```diff
@@ -0,0 +1,200 @@
+--- !'map':HashWithIndifferentAccess
+
+levelsConvert:
```

Comment:
cool this YAML file is pretty readable, so I don't think it will be unreasonable if anyone from worldreader wanted to make minor changes to this.